index.js

```diff
@@ -0,0 +1,24 @@
+const sass = require('node-sass'),
```

Comment:
why this script?

Reply:
It will be a hook so the SASS is compiled and compressed to CSS and we can load that in the client. This solves the SASS compilation on runtime